src/containers/ActivityContainer/Filters.tsx

```diff
@@ -0,0 +1,118 @@
+import CStreamStatus, { StreamStatus } from 'src/components/CStreamStatus';
+import Image from 'next/image';
+
+import FilterModal from './FilterModal';
+import * as Styled from './styles';
+
+import Funnel from 'src/assets/Funnel';
+import searchLogo from 'public/images/search.svg';
+import { useState } from 'react';
+import { IToken } from 'src/reducers/tokens';
+import { IFilterTokens } from 'src/constants/types';
+
+type FiltersProps = {
+  filteredValues: IFilterTokens;
+  setSearchValue: React.Dispatch<React.SetStateAction<string>>;
+  setSelectedStatus: React.Dispatch<React.SetStateAction<StreamStatus>>;
+  setFilteredValues: React.Dispatch<React.SetStateAction<IFilterTokens>>;
```

Comment:
Create a type in models.ts called ReactState and use it anywhere that uses React.Dispatch<React.SetStateAction<T> as a type.

For example

` setSearchValue: ReactState<string> `